scipy/interpolate/src/_fitpackmodule.c

```diff
@@ -191,7 +191,7 @@ fitpack_bispev(PyObject *dummy, PyObject *args)
     ny = ap_ty->dimensions[0];
     mx = ap_x->dimensions[0];
     my = ap_y->dimensions[0];
-    mxy = mx*my;
+    mxy = (unsigned)mx * (unsigned)my;
```

Comment:
Should be `npy_uint64` so it doesn't downcast to 32-bit

Reply:
But what about 32bit devices then?

Reply:
They have 64-bit wide data type (otherwise Numpy doesn't run on them), so it just runs a bit slower. The overflow check on the next line then detects the truncation of the result.

Reply:
@pv fixed

Reply:
unsigned_max doubles is ~35 GB, quite far from unlimited today.